api/HotelsDatabaseContext.cs

```diff
@@ -36,6 +37,29 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                     .IsUnicode(false)
                     .IsFixedLength(true);
             });
+
+            modelBuilder.Entity<Hotel>(entity =>
+            {
+                entity.Property(e => e.Address)
+                    .IsRequired()
+                    .HasMaxLength(155)
+                    .IsUnicode(false);
+
+                entity.Property(e => e.Country)
+                    .IsRequired()
+                    .HasMaxLength(60)
```

Comment:
Why did you choose `60`? Is it the longest name of the country on Earth

Reply:
The longest name is 56, I rounded to 60